integration_test/src/java/com/twitter/heron/integration_test/core/IntegrationTestSpout.java

```diff
@@ -134,12 +134,11 @@ public void nextTuple() {
 
   @Override
   public void ack(Object messageId) {
-    LOG.fine("Received a ack with MessageId: " + messageId);
-
-    tuplesToAck--;
     if (!isTestMessageId(messageId)) {
       delegateSpout.ack(messageId);
     } else {
+      tuplesToAck--;
```

Comment:
Realized this isn't the right change and reverted it, since there are cases where tuples are emitted with a pre-existing message id, in the case that they're not test messages.